.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,65 @@
+name: Run tests
+
+on: push
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+        python-version: [3.5, 3.6, 3.7, 3.8]
+    env:
+      CODACY_PROJECT_TOKEN: ${{ secrets.CODACY_PROJECT_TOKEN }}
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - name: Check coding style
+      run: |
+        pip install flake8 isort
+        flake8 picard --count --show-source --statistics
+        isort --check-only --diff --recursive picard test
+    - name: Test with pytest
+      if: always()
```

Comment:
Again, this should only run if pip install succeeds. Why is it always run?

Reply:
That's intentional, I want to see both the results of the syntax checks and the actual test results. If the syntax checks fails the entire job will still be considered failed, but the tests run anyway. Otherwise you get the issues with your commit reported peu à peu.

Actually this option is something I really like about Github Actions, other CI systems often make achieving this behavior difficult.